test/e2e/proxy_test.go

```diff
@@ -68,7 +70,493 @@ func skipIfProxyDisabled(t *testing.T) {
 func testProxyServiceSessionAffinityCase(t *testing.T, data *TestData) {
 	if len(clusterInfo.podV4NetworkCIDR) != 0 {
 		ipFamily := corev1.IPv4Protocol
-		testProxyServiceSessionAffinity(&ipFamily, []string{"169.254.169.253", "169.254.169.254"}, data, t)
+		testProxyServiceSessionAffinity(&ipFamily, []string{"169.254.169.1", "169.254.169.2"}, data, t)
+	}
+	if len(clusterInfo.podV6NetworkCIDR) != 0 {
+		ipFamily := corev1.IPv6Protocol
+		testProxyServiceSessionAffinity(&ipFamily, []string{"fd75::aabb:ccdd:ef00", "fd75::aabb:ccdd:ef01"}, data, t)
+	}
+}
+
+func skipIfProxyAllDisabled(t *testing.T, data *TestData) {
+	isProxyFull, err := data.IsProxyAll()
+	if err != nil {
+		t.Fatalf("Error getting option antreaProxy.proxyAll value")
+	}
+	if !isProxyFull {
+		t.Skipf("Skipping test because option antreaProxy.proxyAll is not enabled")
+	}
+}
+
+func skipIfKubeProxyEnabledOnLinux(t *testing.T, data *TestData, nodeName string) {
+	pods, err := data.clientset.CoreV1().Pods(antreaNamespace).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		t.Fatalf("Error fetching pods: %v", err)
+	}
+	for _, pod := range pods.Items {
+		if strings.Contains(pod.Name, "kube-proxy") && pod.Spec.NodeName == nodeName {
+			t.Skipf("Skipping test because kube-proxy is running")
+		}
+	}
```

Comment:
Getting kube-proxy DaemonSet directly, instead of listing all Pods and filter by names

Reply:
Thanks, got it.

Reply:
Just **getting** the daemonset with specific name is simpler. And it should use `kubeNamespace` instead of `antreaNamespace`, though they are same for now.